src/assets/data/Assets.types.ts

```diff
@@ -0,0 +1,30 @@
+import {CustomIconNames} from '../../components';
```

Comment:
This import might cause cyclic dependencies in the app. Maybe we should not keep types file for these data here in assets. We can keep them in common folder or maybe create a separate folder for all the types

Reply:
Added a global types file for the assets.